charts/prometheus-node-exporter/templates/daemonset.yaml

```diff
@@ -15,10 +15,15 @@ spec:
   template:
     metadata:
       labels: {{ include "prometheus-node-exporter.labels" . | indent 8 }}
-      {{- if .Values.podAnnotations }}
+      {{- if or  .Values.global.podAnnotations .Values.podAnnotations }}
       annotations:
+      {{- if .Values.global.podAnnotations }}
+      {{- toYaml .Values.global.podAnnotations | nindent 8 }}
```

Comment:
This syntax is inconsistent with all the other modifications above here in the PR. It indents the whole golang template block, and uses nindent and `{{-`. I prefer this syntax. In any case, we should be consistent with our syntaxes. IMHO we should modify all these changes to have the indented template  + nindent + `{{-`